tests/metrics/workload_time/cor_create_time.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2016 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+#  Description of the test:
+#  This test measures the time cc-oci-runtime (cor) takes from creating
+#  a container and before starting it.
+
+set -e
+
+if [ $# -ne 1 ]; then
+  echo -e "Usage: $0 <times to run>" && exit 1
+fi
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "$SCRIPT_PATH/../common/test.common"
+
+TEST_NAME="Cor Start Time"
+LOG_FILE="/run/opencontainer/containers/cc-oci-runtime.log"
+IMAGE='busybox'
+TIMES=$1
+RUNTIME='cor'
+testResultFileNet="$RESULT_DIR/cor-start-time-with-network"
+testResultFileNoNet="$RESULT_DIR/cor-start-time-without-network"
+
+function get_time(){
+	grep "cc-oci-runtime:debug:called as: /usr/bin/cc-oci-runtime create" ${LOG_FILE} | \
```

Comment:
Fixed, I took the `@PACKAGE_NAME@` to get the program name and added a regex to avoid the hard-coded path.